### DISCUSSION TITLE
.NET 6/7 support for WPF

### DISCUSSION THREAD

**Discussion by MZeeshanHaux** [oxyplot/oxyplot#1994] [General]
Hello everyone, I am just wondering if there is a plan for releasing WPF support for .NET 6 or 7 soon?
There is an open [PR](https://github.com/oxyplot/oxyplot/pull/1937) regarding the project update. Is there any update on that?
@VisualMelon @Jonarw

**Comment by VisualMelon** [comment]
The older targets should 'just work' with .NET 6/7; if you have a specific difficultly using the libraries with .NET 6/7, please list it here so we can investigate and prioritise a release if necessary.

**Comment by MZeeshanHaux** [comment]
Thanks @VisualMelon, I will try a new project and add a further discussion if there is any difficulty.
